Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "You know, I recently read about swimming. There are so many species.  But I was attracted the most by scuba diving.
+    m 3eub "Hey [player], have you heard about scuba diving?"
```

Comment:
I don't think this line fits here, especially when you redo the beginning and keep the old introduction here.